File: M5.1 Traductor de pseudo a C y Python/output.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

def main():
    n = input("Cuantas calificaciones: ")
    prom = 0
    i = 0
    while i < n:
        cal = input("Da una calificacion: ")
        prom = prom + cal
        i = i + 1
    prom = prom / n
    if prom > 5:
        print("Aprobado con: " + " " + prom)
    if prom == 10:
        print("Excelente")

if __name__ == "__main__":
    main()

File: M5.1 Traductor de pseudo a C y Python/output.c
#include <stdio.h>

int main(void) {
    printf("Cuantas calificaciones: ");
    scanf("%s", &n);
    prom = 0;
    i = 0;
    while (i < n) {
        printf("Da una calificacion: ");
        scanf("%s", &cal);
        prom = prom + cal;
        i = i + 1;
    }
    prom = prom / n;
    if (prom > 5) {
        printf("%s\n", "Aprobado con: " + " " + prom);
    }
    if (prom == 10) {
        printf("Excelente");
        printf("\n");
    }
return 0;
}
